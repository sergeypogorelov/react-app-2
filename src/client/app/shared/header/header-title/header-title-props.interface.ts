export interface HeaderTitleProps {
  id?: string;
}
