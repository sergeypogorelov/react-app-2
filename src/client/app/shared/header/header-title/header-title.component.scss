@import '../../../../styles/variables';

.header-title {
  font-family: $f-light;
  font-size: 40px;
}
