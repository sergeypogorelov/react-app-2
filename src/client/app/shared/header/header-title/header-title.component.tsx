import './header-title.component.scss';

import React, { FunctionComponent } from 'react';

export const HeaderTitle: FunctionComponent<{}> = (props) => {
  return <h2 className="header-title">{props.children}</h2>;
};
