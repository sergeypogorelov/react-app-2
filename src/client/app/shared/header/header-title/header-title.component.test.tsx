import React from 'react';
import renderer from 'react-test-renderer';

import { HeaderTitle } from './header-title.component';
import { HeaderTitleProps } from './header-title-props.interface';

test('HeaderTitle component should render correctly', () => {
  const props: HeaderTitleProps = {
    id: 'id',
  };
  const tree = renderer
    .create(<HeaderTitle {...props}>test</HeaderTitle>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
