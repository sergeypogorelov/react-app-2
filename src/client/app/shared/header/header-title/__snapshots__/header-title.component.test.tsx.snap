// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle component should render correctly 1`] = `
<h2
  className="header-title"
  id="id"
>
  test
</h2>
`;
